keps/sig-storage/20190709-csi-snapshot.md

```diff
@@ -119,12 +244,14 @@ There are other things we are working on for Beta:
     labels:
     * operation_name, i.e., creation-snapshot, delete-snapshot
     * csi-driver-name
-* Update snapshot CRDs to v1beta1 and enable VolumeSnapshotDataSource feature gate by default.
 
 ## Implementation History
 
-Volume snapshot is implemented as alpha feature in this repo in Kubernetes v1.12:
+Volume snapshot is implemented as alpha feature in this repo in Kubernetes v1.12 and is promoted to beta in Kubernetes 1.17:
```

Comment:
I would probably just structure this section as:

```
K8s 1.12: alpha
K8s 1.17: beta
```

I'm not sure we need the actual feature gate PRs here

Reply:
Updated.